python-frontend/src/test/java/org/sonar/python/quickfix/PythonQuickFixTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.quickfix;
+
+import java.util.Arrays;
+import org.junit.Test;
+import org.sonar.plugins.python.api.IssueLocation;
+import org.sonar.plugins.python.api.LocationInFile;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PythonQuickFixTest {
+
+  @Test
+  public void test() {
+    PythonQuickFix.Builder builder = PythonQuickFix.newQuickFix("New quickfix");
+    String message = "This is a replacement text";
```

Comment:
Lets' rename `message` to something like `replacementText`, it would be less confusing.